api/src/pcapi/core/finance/api.py

```diff
@@ -2001,7 +1986,7 @@ def _get_cashflows_by_bank_accounts(batch: models.CashflowBatch, only_debit_note
         )
     ).filter(models.Cashflow.batchId == batch.id)
 
-    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount < 0)
+    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount < 1)
```

Comment:
```suggestion
    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount =< 0)
```
C'est plus clair, je trouve (sinon on se demande pourquoi 1)